game-core/src/main/java/tools/image/TileImageReconstructor.java

```diff
@@ -34,23 +35,30 @@
 /**
  * For taking a folder of basetiles and putting them back together into an image.
  */
-public class TileImageReconstructor {
-  private static String baseTileLocation = null;
-  private static String imageSaveLocation = null;
-  private static File mapFolderLocation = null;
+public final class TileImageReconstructor {
```

Comment:
Why are classes being marked as final?

Reply:
I made the constructors private to avoid someone misusing this class (e.g. instantiating it off the EDT).  Making the class `final` was the next natural step.

See also: _Effective Java_, Item 19: Design and document for inheritance or else prohibit it.